code/modules/mob/mob.dm

```diff
@@ -1593,3 +1593,24 @@ GLOBAL_LIST_INIT(holy_areas, typecacheof(list(
 	else
 		. = invoked_callback.Invoke()
 	usr = temp
+
+/mob/verb/give_kudos(mob/living/target as mob in oview())
+	set category = null
+	set name = "Give Kudos (OOC)"
+
+
+	if(target == src)
+		to_chat(src, "<span class='warning'>You cannot give kudos to yourself!</span>")
+		return
+
+	to_chat(src, "<span class='notice'>You've given kudos to [target]!</span>")
```

Comment:
Probably a good idea to have a message if you've already given them kudos

Reply:
I had that originally, but I was thinking that it might open the door for some light metagaming since it can only truly be tracked on minds. If you don't know if a mob has a mind or not, just give them kudos and you'll find out.
This might not be a true issue, but it's what I was thinking, at least.